app/models/certification.rb

```diff
@@ -0,0 +1,42 @@
+class Certification < ApplicationRecord
```

Comment:
I don't get the distinction between certificate and certification. I understand the first is the token and the second is the process, but they are even synonyms according to some sources: https://www.merriam-webster.com/thesaurus/certification

Reply:
I think I'd prefer the term `Diploma` instead of `Certificate`

Reply:
Alternatively, if changing `Certificate => Diploma`  is a too expensive rename, we could instead rename `Certification` to `Accreditation`: https://www.synonym.com/synonyms/accreditation

Reply:
Yep, we were mislead by spanish translation of those words. It was meant to be the difference between "la certificación" y "los certificados emitidos".
`Accreditation` and `Certificate` seem to be a good fit for what we want.

Reply:
Reading this...
> Certification generally refers to an earned credential that demonstrates the holder's specialized knowledge, skills, and experience. Certification differs from a certificate program, which is usually an educational offering that confers a document at the program's conclusion.

I think that `Certificate` is a good name and we should rename `Certifications` by `CertificateProgram`

Reply:
Maybe `CertificateProgram` is a good term. But if it the case, I am confused. What do we want to represent here? 

An `Accreditation` and a `CertificateProgram` is not the same, one refers to the precise moment where examination takes places, and the other to a whole educative process of training and maybe examination. What do you want to represent here @fedescarpa with this class? Is this a single object we create _a priori_ when the project starts, or are we going to create multiple instances for it during its life-cycle? Which will be it's `created_at`?